tests/unit/lms/services/lti_role_service_test.py

```diff
@@ -22,7 +23,10 @@ def test_get_roles(self, svc, existing_roles):
         role_descriptions.append(existing_roles[0].value)
         role_descriptions.extend(new_roles)
 
-        roles = svc.get_roles(", ".join(role_descriptions))
+        if roles_as_string:
+            # pylint:  disable=redefined-variable-type
```

Comment:
Yes, we are redefining the type of a variable here pylint, but that's the point here so adding a `disable`